windows/src/main/csharp/ch/cyberduck/ui/core/preferences/PackagedDataMigrator.cs

```diff
@@ -0,0 +1,100 @@
+﻿using ch.cyberduck.core.preferences;
+using Ch.Cyberduck.Core.Preferences;
+using org.apache.logging.log4j;
+using System;
+using System.IO;
+using Windows.Win32.UI.Shell;
+using static Windows.Win32.CorePInvoke;
+using static Windows.Win32.PInvoke;
+
+namespace Ch.Cyberduck.Core.Ui.Preferences;
+
+internal static class PackagedDataMigrator
+{
+    public static void Migrate()
+    {
+        // Logger is used for short time.
+        // Don't need to keep an instance alive longer than necessary
+        var logger = LogManager.getLogger(typeof(PackagedDataMigrator).FullName);
```

Comment:
Garbage collector will do its job instead of passing as parameter?

Reply:
I don't want to have the logger hanging around straining Gen2 (long living objects).
Same applies to java: static fields are _never_ collected by GC.

Reply:
```
static class A
{
    static readonly object B = new();
}
```

B won't be collected until application shutdown.